mitodl/mit-open
# Do not ingest prolearn courses/programs from the past

### What are the relevant tickets?
Closes https://github.com/mitodl/hq/issues/4362

### Description (What does it do?)
Adjusts the prolearn pipeline to skip over any courses that don't have a start date in the future.


### How can this be tested?
- On the main branch, run `./manage.py backpopulate_prolearn_data`.  Then go to https://mitopen.odl.mit.edu/search?offered_by=see&sortby=start_date - you will see courses with start dates in the past.
- Switch to this branch, restart containers, and repeat the above.  This time the courses with past start dates should not show up.
